packages/lib/src/components/internal/SecuredFields/SFP/types.ts

```diff
@@ -0,0 +1,24 @@
+import { CVCPolicyType, DatePolicyType } from '../lib/types';
+import { AddressData } from '../../../../types';
+
+export interface SFPState {
+    status?: string;
+    brand?: string;
+    errors?: object;
+    valid: object;
+    data: object;
```

Comment:
Is it possible to add proper type instead of `object` for these properties? Or is it kinda complex/dynamic objects?

Reply:
They are complex-ish objects. I've typed one in another PR so I'll port that over. The rest I'll deal with in a subsequent PR